lib/ClangImporter/ImportDecl.cpp

```diff
@@ -816,6 +816,31 @@ static bool isPrintLikeMethod(DeclName name, const DeclContext *dc) {
 using MirroredMethodEntry =
   std::tuple<const clang::ObjCMethodDecl*, ProtocolDecl*, bool /*isAsync*/>;
 
+ImportedType tryImportOptionsTypeForField(const clang::QualType type,
```

Comment:
I know we use this pattern elsewhere, I thought we already had a helper function for this. Could we just update that? 

If not, I don't think we need to say "for field". Also, maybe we can match a similar function and say "as option set type". So maybe `findOptionSetType` or `findOptionSetTypeForTypedef`.

Reply:
I don't think we have a helper function, I grepped everywhere  in ClangImporter that we use `findAnonymousEnumForTypedef` to try and find it but I dont see one. I can make all the other locations that follow the pattern use this helper and drop the `ForField` though.